scripts/fast-reboot

```diff
@@ -416,6 +416,10 @@ function setup_reboot_variables()
         local fstype=$(blkid -o value -s TYPE ${sonic_dev})
         BOOT_OPTIONS="${BOOT_OPTIONS} ssd-upgrader-part=${sonic_dev},${fstype}"
     fi
+
+    # Set governor to performance to speed up boot process.
+    # The governor is reset back to kernel default in warmboot-finalizer script.
+    BOOT_OPTIONS="${BOOT_OPTIONS} cpufreq.default_governor=performance"
```

Comment:
@stepanblyschak let's add another check here:
`/sys/devices/system/cpu/cpu0/cpufreq/scaling_available_governors`
to make sure `performance` is really supported by CPU.
If not - log a relevant message.

Reply:
@nazariig governor is not a cpu feature, governor talks to the scaling driver (cpu specific) to set a target.  scaling_available_governors  - This list appears to be formed from the global cpufreq_governor_list in the cpufreq driver, where all governors that are included in the kernel are registered.

This cmdline param is fairly new (>5.9) but it is very appealing to use in this case (no need to wait for user space and sysfs to be available to set the governor from user space).

When this won't work - if the next kernel is not built with performance governor as a builtin driver it won't work (governor will not change).
For that, **we would have to extract the kernel config from the next image rather then checking current image scaling_available_governors**. And I generally don't think we would ever remove features from kernel in next releases.